src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,15 @@
-import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList = ({ ...moviesObject }) => (
+  <div className="movies">
+    {moviesObject.movies.map(
+      (movie: Movie) => {
+        return (
+          <MovieCard {...movie} key={movie.imdbId} />
```

Comment:
Maybe test are failing because of this? Try
```suggestion
          <MovieCard movie={movie} key={movie.imdbId} />
```

Reply:
I added movie attribute.